app/src/cli/commands/clone.ts

```diff
@@ -0,0 +1,47 @@
+import * as QueryString from 'querystring'
+import { URL } from 'url'
+
+import { CommandError } from '../util'
+import { openDesktop } from '../open-desktop'
+import { ICommandModule, mriArgv } from '../load-commands'
+
+interface ICloneArgs extends mriArgv {
+  readonly branch?: string
+}
+
+const command: ICommandModule = {
+  command: 'clone <url|slug>',
+  description: 'Clone a repository',
+  args: [
+    {
+      name: 'url|slug',
+      required: true,
+      description: 'The URL to clone, or the GitHub repo slug to clone',
```

Comment:
I'm not sure `slug` is clear enough here (unless other tools use this - I checked `hub`'s [docs](https://github.com/github/hub/blob/a45f4573a32e346d7046b47657263cd17cb60c9e/commands/clone.go#L22-L23) and they didn't use anything).

What about something like this:

> or the GitHub owner/name alias to clone

Reply:
✅